packages/components/src/Form/Input.tsx

```diff
@@ -5,11 +5,14 @@ interface InputProps {
   width?: string;
   value: string;
   disabled?: boolean;
-  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onFocus?: (event: React.FocusEvent<HTMLInputElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
```

Comment:
The interface of blur event is `FocusEvent`
https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event